packages/ts-client/src/webrtc/tracks/transceivers.ts

```diff
@@ -25,12 +25,19 @@ const createVideoTransceiverConfig = (
 ): RTCRtpTransceiverInit => {
   if (!trackContext.simulcastConfig) throw new Error(`Simulcast config for track ${trackContext.trackId} not found.`);
 
-  if (typeof maxBandwidth !== 'number' && trackContext.simulcastConfig.enabled) {
-    return createSimulcastTransceiverConfig(trackContext, maxBandwidth);
+  if (typeof maxBandwidth === 'number' && maxBandwidth !== 0) {
+    return createNonSimulcastTransceiverConfig(trackContext, maxBandwidth);
```

Comment:
If I understand correctly, this `if` prevents the creation of a transceiver config for a non-simulcast track with unlimited bandwidth, which is acceptable